build/media-suite_helper.sh

```diff
@@ -2013,6 +2013,15 @@ create_cmake_toolchain() {
         printf '%s\n' "${toolchain_file[@]}" > "$LOCALDESTDIR"/etc/toolchain.cmake
 }
 
+get_signature() {
+    # get_signature 96865171 0F3BE490
+    # adds keys to gpg keychain for verifying
+    {
+        gpg -q --receive-keys "$@" ||
+            gpg -q --keyserver pgp.mit.edu --receive-keys "$@"
```

Comment:
Any reason for this keyserver instead of the newer `hkps://keys.openpgp.org`?

Reply:
None, just because I wasn't sure what keyserver gpg doesn't use by default other than sks and I hope I can trust mit to keep their server up

Reply:
Yeah, but I'd rather try to use a newer keyserver which is not vulnerable to the key poisoning stuff.

Reply:
Changed to first try `hkps://keys.openpgp.org`, then `hkps://hkps.pool.sks-keyservers.net`, then `hkps://pgp.mit.edu` in case the keys we need are not yet in `hkps://keys.openpgp.org` with the default server being `hkps://keys.openpgp.org`